src/pages/patientView/clinicalInformation/ClinicalInformationMutationalSignatureTable.tsx

```diff
@@ -83,13 +96,47 @@ export default class ClinicalInformationMutationalSignatureTable extends React.C
     @computed get tableData() {
         return prepareMutationalSignatureDataForTable(this.props.data);
     }
+    @computed get tooltipInfo() {
+        return (
+            <div
+                style={{ maxWidth: 450 }}
+                data-test="SignificantMutationalSignaturesTooltip"
+            >
+                <div>
+                    <h4>
+                        <b>Signature:</b>
+                        {this.props.signature}
+                    </h4>
+                    <p style={{ fontSize: '16px' }}>
```

Comment:
is there reason why you are making font bigger here?  wold be best to leave it in the standard size

Reply:
Removed `style={{fontSize:'16px'}}`